hummingbot/connector/derivative/coinflex_perpetual/coinflex_perpetual_api_order_book_data_source.py

```diff
@@ -0,0 +1,518 @@
+                                     api_factory: Optional[WebAssistantsFactory] = None,
+                                     throttler: Optional[AsyncThrottler] = None) -> Dict[str, float]:
+        """
+        Return a dictionary the trading_pair as key and the current price as value for each trading pair passed as
+        parameter
+        :param trading_pairs: list of trading pairs to get the prices for
+        :param domain: which CoinFLEX domain we are connecting to (the default value is 'com')
+        :param api_factory: the instance of the web assistant factory to be used when doing requests to the server.
+        If no instance is provided then a new one will be created.
+        :param throttler: the instance of the throttler to use to limit request to the server. If it is not specified
+        the function will create a new one.
+        :return: Dictionary of associations between token pair and its latest price
+        """
+
+        resp = await web_utils.api_request(
+            path=CONSTANTS.TICKER_PRICE_CHANGE_URL,
+            api_factory=api_factory,
+            throttler=throttler,
+            domain=domain,
+            method=RESTMethod.GET,
+        )
+
+        results = {}
+
+        for t_pair in trading_pairs:
+            symbol = await cls.convert_to_exchange_trading_pair(
+                hb_trading_pair=t_pair,
+                domain=domain,
+                throttler=throttler)
+            matched_ticker = [t for t in resp if t.get("marketCode") == symbol]
+            if not (len(matched_ticker) and "last" in matched_ticker[0]):
+                raise IOError(f"Error fetching last traded prices for {t_pair}. "
+                              f"Response: {resp}.")
+            results[t_pair] = float(matched_ticker[0]["last"])
+
+        return results
+
+    @classmethod
+    async def get_last_traded_price(cls,
+                                    trading_pair: str,
+                                    domain: str = CONSTANTS.DEFAULT_DOMAIN,
+                                    api_factory: Optional[WebAssistantsFactory] = None,
+                                    throttler: Optional[AsyncThrottler] = None) -> float:
+
+        symbol = await cls.convert_to_exchange_trading_pair(
+            hb_trading_pair=trading_pair,
+            domain=domain,
+            throttler=throttler)
+
+        resp = await web_utils.api_request(
+            path=CONSTANTS.TICKER_PRICE_CHANGE_URL,
+            api_factory=api_factory,
+            throttler=throttler,
+            domain=domain,
+            method=RESTMethod.GET,
+        )
+
+        matched_ticker = [t for t in resp if t.get("marketCode") == symbol]
+        if not (len(matched_ticker) and "last" in matched_ticker[0]):
+            raise IOError(f"Error fetching last traded prices for {trading_pair}. "
+                          f"Response: {resp}.")
+        return float(matched_ticker[0]["last"])
```

Comment:
This method does not seem to be called anywhere. If it is not used it should be removed.